src/containers/CreatePipelineRun/YAMLEditor.js

```diff
@@ -0,0 +1,245 @@
+      setState(state => ({
+        ...state,
+        validationError: true,
+        validationErrorMessage: validationResult.message
+      }));
+      return;
+    }
+
+    let obj;
+    try {
+      obj = yaml.load(code);
+    } catch (e) {
+      setState(state => ({
+        ...state,
+        validationError: true,
+        validationErrorMessage: e.message
+      }));
+      return;
+    }
+
+    validationResult = validateNamespace(obj);
+    if (validationResult && !validationResult.valid) {
+      setState(state => ({
+        ...state,
+        validationError: true,
+        validationErrorMessage: validationResult.message
+      }));
+      return;
+    }
+
+    setState(state => ({ ...state, creating: true }));
+    const nspace = obj?.metadata?.namespace;
+
+    createPipelineRunRaw({
+      namespace: nspace,
+      payload: obj
+    })
+      .then(() => {
+        navigate(urls.pipelineRuns.byNamespace({ namespace: nspace }));
+      })
+      .catch(error => {
+        error.response.text().then(text => {
+          const statusCode = error.response.status;
+          let errorMessage = `error code ${statusCode}`;
+          if (text) {
+            errorMessage = `${text} (error code ${statusCode})`;
+          }
+          setState(state => ({
+            ...state,
+            creating: false,
+            submitError: errorMessage
+          }));
+        });
+      });
+  }
+
+  function onChange(newValue, _) {
+    setState(state => ({
+      ...state,
+      code: newValue
+    }));
+  }
+
+  function resetError() {
+    setState(state => ({ ...state, submitError: '' }));
+  }
+
+  function handleClose() {
+    let url = urls.pipelineRuns.all();
+    if (selectedNamespace && selectedNamespace !== ALL_NAMESPACES) {
+      url = urls.pipelineRuns.byNamespace({ namespace: selectedNamespace });
+    }
+    navigate(url);
+  }
+
+  return (
+    <div className="tkn--create">
+      <div className="tkn--create--heading">
+        <h1 id="main-content-header">
+          {intl.formatMessage({
+            id: 'dashboard.createPipelineRun.title',
+            defaultMessage: 'Create PipelineRun'
+          })}
+        </h1>
+      </div>
+      <Form>
+        {pipelineError && (
+          <InlineNotification
+            kind="error"
+            title={intl.formatMessage({
+              id: 'dashboard.createPipelineRun.errorLoading',
+              defaultMessage: 'Error retrieving Pipeline information'
+            })}
+            lowContrast
+          />
+        )}
+        {validationError && (
+          <InlineNotification
+            kind="error"
+            title={intl.formatMessage({
+              id: 'dashboard.createRun.yaml.validationError',
+              defaultMessage: 'Please fix errors, then resubmit'
+            })}
+            subtitle={validationErrorMessage}
+            lowContrast
+          />
+        )}
+        {submitError !== '' && (
```

Comment:
```suggestion
        {submitError && (
```

`''` is falsy, consistency with other checks in this code